_projects/area_indicators.md

```diff
@@ -0,0 +1,60 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Scala programming experience"
+  - "Python programming experience not required but preferred"
+  - "NetCDF experience not required but preferred"
```

Comment:
"Requirements" is a scary section that will have people fall off the ship too early. Scala is already narrows the pool a lot, though I agree it the most necessary of the skillsets.

- Do you think we can broaden it to "functional programming experience (haskell, scala)"? A college friend who graduated CS learned some Haskell in school, so I have reason to believe that we could broaden our prospects that way. The project will still be written in scala.

- netCDF is not too common a dataset to work with, particularly I think for a student. I would like to push this out of requirements, it will be something the fellow will expect to learn, or list it as "bonus" skills like in our regular job posts. Is there a "bonus skills" table column ?

Reply:
Ultimately some of this comes down to the mentors I think - how confident are folks that they can teach Scala to someone, and what transferable skills do they think fellows should have if they lack Scala experience?

Perhaps this would work? 
```markdown
requirements: # bullet list of requirements – one requirement per line – follow below format
  - "Functional programming experience required (Haskell, Scala, ML, etc.) "
  - "Scala programming experience not required but preferred"
  - "Python programming experience not required but preferred"
  - "NetCDF experience a bonus"
```

Reply:
Me! I'm confident we can teach scala in a short timeframe! I'm with Jenny though about the requirements -- I don't think we should worry too much about language, and could broaden to something like:

- familiarity building or working with APIs
- interest in scala / fp

If the project had to be written from scratch, I'd be worried about how to teach, but with the existing prototype we have plenty of syntax examples and can use them to teach about project structure

Reply:
I reworked this section in https://github.com/azavea/fellowship-program-website/pull/66/commits/0b5150a0412b45dd8677aeffa35edd2b5f3195a3